versions.tfvars.json

```diff
@@ -44,17 +44,17 @@
     "public.ecr.aws/eks-distro/kubernetes-csi/livenessprobe":         "v2.2.0-eks-1-18-13",
     "public.ecr.aws/eks-distro/kubernetes-csi/node-driver-registrar": "v2.1.0-eks-1-18-13",
     "public.ecr.aws/eks-distro/kubernetes-csi/external-provisioner":  "v2.1.1-eks-1-18-13",
```

Comment:
Couldn't it be a good idea to also update the other images?

Reply:
We have to update the Helm charts first, so we made a different issue for the remaining images.

Reply:
Ok